docs/rules/vue-essential/vif-with-vfor.md

```diff
@@ -14,13 +14,13 @@ Checks if the component is using `v-if` with `v-for`. &nbsp;&nbsp;<br />
 The following code uses `v-if` together with `v-for`, which is not recommended as it can lead to performance issues and inefficient rendering.
 :::
 
-```vue
+```js
 <template>
   <ul>
     <li
       v-for="user in users"
-      v-if="user.isActive"
-      :key="user.id"
+      v-if="user.isActive" // [!code warning]
+      v-bind:key="user.id"
```

Comment:
if code has `v-bind` it will fire the shorthand rule.

It should be `:key="user.id"`

Reply:
Oh that was a workaround to use the `[!code warning]. It gets messy the because of the `:`.

I'll change it

Reply:
fixed it in last commit